07_train_final_classifiers/collect_full_models.ipynb

```diff
@@ -0,0 +1,706 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    coefs[gene][coefs[gene][gene].isna()].head()
</pre><p>If you are expecting all these to be NaN, then you can also add another assert:</p><pre class="ql-syntax" spellcheck="false">_tmp = coefs[gene][coefs[gene][gene].isna()] # should only be on head() ?
display(_tmp)
assert _tmp.isna().all()
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/mpmp/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>